rook/main.py

```diff
@@ -68,6 +68,8 @@ class NoColors:
 parser = argparse.ArgumentParser(description="Test Runner")
 parser.add_argument('-j', '--jobs', dest='number_jobs', type=int, default=1,
                     help='Specifies number of tests to run simultaneously (default: 1)')
+parser.add_argument('--longest-jobs',dest='longest_jobs', type=int, default=0,
+                    help='Only for compatibility')
```

Comment:
Thanks for add this option, is it already used in our regression testing?

Reply:
Civet uses it to "Print the longest running jobs upon completion" We could add an actual feature to do this if we want.